seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/SqlType.java

```diff
@@ -41,5 +41,6 @@ public enum SqlType {
     BFLOAT16_VECTOR,
     SPARSE_FLOAT_VECTOR,
     ROW,
-    MULTIPLE_ROW;
+    MULTIPLE_ROW,
+    INET;
```

Comment:
It seems to me that there is no need for `INET`, which can be transferred in `string` with ST

Reply:
+1

Reply:
cc @Hisoka-X

Reply:
Seatunnel built-in data types require a high degree of abstraction

Reply:
@hailin0 @corgy-w When  for sink，pg table  also uses the inet type，this cannot be written

Reply:
and for source，When the schema is automatically obtained，PostgresTypeConverter class does not support the data type "inet"，so an error was reported before transform

Reply:
and for source,when  the schema is automatically obtained， PostgresTypeConverter class does not support the data type "inet"，so an error was reported before transform.
![Uploading image.png…]()

Reply:
We can increase the pg base type, but we don't need to increase the ST base type @CosmosNi 
![image](https://github.com/user-attachments/assets/c1595b63-b37d-4cec-b531-c731b4e0bc9b)

Reply:
@corgy-w your idea is good.I'll see if I can solve this problem through configuration.

Reply:
> It seems to me that there is no need for INET, which can be transferred in string with ST

+1